File: year2020/month09/day0918/ms_0918.cs
using System;

namespace algorithm_60
{
    class ms_0918
    {
        static void AVC(int[] X, double[] Y) // Absolute Value Calculation
        {
            Console.WriteLine((Math.Abs(X[0]) > Math.Abs(X[1])? X[0] : X[1]));
            Console.WriteLine((Math.Abs(Y[0]) > Math.Abs(Y[1])? Y[1] : Y[0]));
        }

        static void Main(string[] args)
        {
            string[] input = Console.ReadLine().Split(' ');
            int[] intData = new int[2];

            for(int i = 0; i < intData.Length; i++)
                intData[i] = Convert.ToInt32(input[i]);
            
            input = Console.ReadLine().Split(' ');
            double[] doubleData = new double[2];

            for(int i = 0; i < doubleData.Length; i++)
                doubleData[i] = Convert.ToDouble(input[i]);

            AVC(intData, doubleData);
        }
    }
}

File: year2020/month09/day0918/ms_0918.cpp
#include <iostream>

using namespace std;

void AVC(int* X, double* Y); // Absolute Value Calculation

int main() {
	int iData[2]; // int type data
	double dData[2]; // double type data

	for (int i = 0; i < 2; i++)
		scanf_s("%d", &iData[i]);

	for (int i = 0; i < 2; i++)
		scanf_s("%lf", &dData[i]);

	AVC(iData, dData);

	return 0;
}

void AVC(int* X, double* Y) {
	if (abs(*(X + 0)) > abs(*(X + 1)))
		cout << *X;
	else
		cout << *(++X);

	cout << endl;

	if (abs(*(Y + 0)) > abs(*(Y + 1)))
		cout << *(++Y);
	else
		cout << *Y;
}